wcag-ruleset-review-process.md

```diff
@@ -45,7 +45,7 @@ A facilitator of the ACT Task Force creates a survey for the rule, in which Task
 
 This survey will be open for at least 5 work days, to give ACT Task Force members sufficient time to review the rule. No more than 5 rule surveys will be due in the same week.
 
-If the rule does not pass this step, the liaison has 5 work days to put the feedback in the issue for the proposal using the template in appendix 1. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
+If the rule does not pass this step, the liaison has 5 work days to put the feedback in the issue for the proposal using the template in appendix 1. When submitting the feedback, the liaison must notify the person who opened the issue and ask when the feedback can be resolved. If changes are editorial in nature, and the rule provider can resolve those within 5 work days, the rule will be surveyed again straight away. If not, the proposal will be closed by a TF facilitator. Proposals can always be resubmitted, which will restart the process from step 1. This puts the proposal on the bottom of the backlog.
```

Comment:
So is this a change in the way we're currently handling survey results? "the liaison has 5 work days to put the feedback in the issue for the proposal using the template in appendix 1."  Currently we open a new issue, reference the previous issue the survey was on and use the template to fill in the comments. This is fine, but I want to make sure.  I actually like this because the issue maintenance on our side was getting a bit hard to follow.

Perhaps the reference to the template in Appendix 1 could be a link to the appendix for easier reference.

Assuming "liaison" is the ACT TF liaison. May want to clarify that.  

I like Kathy's suggestions for edits.